toolchain/check/testdata/builtins/int/left_shift.carbon

```diff
@@ -2,7 +2,7 @@
 // Exceptions. See /LICENSE for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
-// EXTRA-ARGS: --no-dump-sem-ir
+// EXTRA-ARGS: --dump-sem-ir-ranges=only
```

Comment:
This test file's check lines seem to be a fair bit longer than some of the others that went from no-dump-sem-ir to ranges=only. Perhaps it could use some more begin/ends?

I'm probably reading something wrong - but it /looks/ like most of the output is from i32.carbon, which has the usual begin/end pair around the relevant operation. So I'm not sure why this'd have so much more output than other similar tests.

Reply:
True it's longer, but this is expected. Note there's only two files getting output. There are just more specifics.

Since this is "only" mode, adding more begin/ends can only increase the amount of output -- it wouldn't decrease.